indico/modules/events/editing/controllers/base.py

```diff
@@ -7,30 +7,65 @@
 
 from __future__ import unicode_literals
 
+import re
+
 from flask import request, session
+from werkzeug.exceptions import Unauthorized
 
 from indico.modules.events.contributions.controllers.display import RHContributionDisplayBase
 from indico.modules.events.controllers.base import RHDisplayEventBase
 from indico.modules.events.editing.models.editable import Editable, EditableType
+from indico.modules.events.editing.settings import editing_settings
 from indico.modules.events.management.controllers.base import RHManageEventBase
 from indico.web.rh import RequireUserMixin
 
 
-class RHEditingBase(RequireUserMixin, RHDisplayEventBase):
+AUTH_BEARER_RE = re.compile(r"Bearer ([\w-]+)")
```

Comment:
```suggestion
AUTH_BEARER_RE = re.compile(r'Bearer ([\w-]+)')
```

Reply:
Can't wait to start using Black.

Reply:
unfortunately black cannot enforce single quotes :/

Reply:
No way!